test/core/test_html_structure.rb

```diff
@@ -7,7 +7,7 @@ def test_simple_render
 html
   head
     title Simple Test Title
-  body 
```

Comment:
Please leave this trailing space, it's actually a test case for #82.

It'd be great if you also could add a line like

```ruby
assert_match source, /body\s+$/, "missing test case for issue #82"
```